Factory/RequestFactory.php

```diff
@@ -50,6 +52,12 @@ public function createOne(array $tokens, $apiUrl, array $options) {
         $query   = HttpQuery::create($tokens);
         $payload = $method === HttpMethods::GET || $method === HttpMethods::DELETE ? null : Payload::create($tokens);
 
+        $options['CURLOPT_HTTPHEADER'] = array_merge(
```

Comment:
Ok now I am confused, too.

Let's make it clear:
HttpHeader.create:
(array $options, array $tokens) -> Options

```typedef Options {
    CURLOPT_HTTPHEADER: array
}
```